src/mod/endpoints/mod_verto/mod_verto.c

```diff
@@ -1764,41 +1764,52 @@ static void http_run(jsock_t *jsock)
 		goto done;
 	}
 
-	if (!strncmp(request->method, "POST", 4) && request->content_length && request->content_type &&
-		!strncmp(request->content_type, "application/x-www-form-urlencoded", 33)) {
-
-		char *buffer = NULL;
-		switch_ssize_t len = 0, bytes = 0;
-
-		if (request->content_length && request->content_length > 10 * 1024 * 1024 - 1) {
-			char *data = "HTTP/1.1 413 Request Entity Too Large\r\n"
-				"Content-Length: 0\r\n\r\n";
-			kws_raw_write(jsock->ws, data, strlen(data));
-			request->keepalive = 0;
-			goto done;
-		}
+	if (!strncmp(request->method, "POST", 4) && request->content_length && request->content_type) {
+		switch_bool_t content_type_urlencoded = SWITCH_FALSE;
+		switch_bool_t content_type_json = SWITCH_FALSE;
+
+		content_type_urlencoded = !strncmp(request->content_type, "application/x-www-form-urlencoded", 33);
+		content_type_json = !strncmp(request->content_type, "application/json", 16);
+
+		if (content_type_urlencoded || content_type_json) {
+			char *buffer = NULL;
+			switch_ssize_t len = 0, bytes = 0;
+
+			if (request->content_length && request->content_length > 10 * 1024 * 1024 - 1) {
```

Comment:
This is redundant. `request->content_length` is already checked.

Reply:
Fixed in the latest commit.